src/WrapperApp/WrapperApp.tsx

```diff
@@ -51,6 +52,10 @@ function WrapperApp() {
 		if (isAuthorized && tabsValue === 'login') setTabsValue('editor');
 	}, [isAuthorized, tabsValue]);
 
+	useEffect(() => {
+		document.title = camelCaseToNormalText(tabsValue); //e.g. we've got 'inputFiles' as a value of tabsValue and this function converts this value to 'Inputs File'
```

Comment:
```suggestion
		document.title = camelCaseToNormalText(tabsValue); //e.g. we've got 'inputFiles' as a value of tabsValue and this function converts this value to 'Input Files'
```

does it convert to `Inputs File` or to `Input Files` ?